torch/_torch_docs.py

```diff
@@ -4343,6 +4343,48 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(self, other, *, out=None) -> Tensor
+
+Computes ``self * log(other)`` with the following cases.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        \text{NaN} & \text{if } \text{other}_{i} = \text{NaN} \\
+        0 & \text{if } \text{self}_{i} = 0.0 \\
+        self_i * \log{(other_i)} & \text{otherwise}
```

Comment:
`self_i` -> `\text{self}_i` and `other_i` -> `\text{other}_i` on this line

And one last change is that "self" should be renamed to "input" in this doc, sorry I forgot to mention that earlier. Just like how the parameters are named in add:

https://pytorch.org/docs/master/generated/torch.add.html?highlight=add#torch.add